test/ttmlir/Silicon/TTNN/simple_broadcast.mlir

```diff
@@ -0,0 +1,23 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path%" %s > %t.mlir
+// RUN: FileCheck %s --input-file=%t.mlir
+// RUN: ttmlir-translate --ttnn-to-flatbuffer %t.mlir > %t.ttnn
+#any_device = #tt.operand_constraint<dram|l1|tile|any_device|any_device_tile>
+
+func.func public @broadcast(%arg0: tensor<32x32xf32>) -> (tensor<32x32xf32>) {
+  %0 = "ttir.constant"() <{value = dense<0xFF800000> : tensor<1xf32>}> : () -> tensor<1xf32>
+  %1 = tensor.empty() : tensor<32x1xf32>
+  %2 = "ttir.max"(%arg0, %1) <{dim_arg = [1 : i32], keep_dim = true, operand_constraints = [#tt.operand_constraint<dram|l1|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device_tile>, #tt.operand_constraint<dram|l1|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device_tile>]}> : (tensor<32x32xf32>, tensor<32x1xf32>) -> tensor<32x1xf32>
```

Comment:
nit: use `#any_device` constraint from the top of the file instead of writing all these constraints in each op